.github/workflows/test.yml

```diff
@@ -12,20 +12,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:  # https://docs.djangoproject.com/en/5.0/faq/install/#faq-python-version-support
-        django-version: [32, 41, 42, 50]
-        python-version: [3.8, 3.9, "3.10", 3.11, 3.12]
-
-        exclude:
-          - django-version: 32
-            python-version: 3.11
-          - django-version: 32
-            python-version: 3.12
-          - django-version: 41
-            python-version: 3.12
-          - django-version: 50
-            python-version: 3.8
-          - django-version: 50
-            python-version: 3.9
+        django-version: [50]
+        python-version: ["3.10", 3.11, 3.12]
```

Comment:
Let's leave Django 4.1 and Python 3.9 in here, as these are still in security / LTS support.

Reply:
I double checked and looks like it's 4.2 the LTS one according to https://www.djangoproject.com/download/
I readded it with py3.9 as minimal python runtime

Reply:
Ah yep, Django 4.2 is the one I meant, sorry :-)